product_docs/docs/edb_plus/40/03_installing_edb_plus/install_on_linux/x86_amd64/07_edbplus_ubuntu20_deb10_x86.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Installing EDB*Plus on Ubuntu 20.04/Debian 10 x86"
+navTitle: "Ubuntu 20.04/Debian 10"
+---
+
+To install EDB\*Plus on a Debian or Ubuntu host, you must have credentials that allow access to the EDB repository. To request credentials for the repository, visit:
+
+ <https://www.enterprisedb.com/repository-access-request>
+
+The following steps will walk you through using the EDB apt repository to install a debian package. When using the commands, replace the `username` and `password` with the credentials provided by EDB.
+
+1. Assume superuser privileges:
+    ```text
+    sudo su -
+    ```
+
+1. Configure the EDB repository:
+
+    On Ubuntu 20:
+    ```text
+    sh -c 'echo "deb https://USERNAME:PASSWORD@apt.enterprisedb.com/$(lsb_release -cs)-edb/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/edb-$(lsb_release -cs).list'
+    ```
+    On Debian 10:
+    
+    a.  Set up the EDB repository:
+        ```text
```

Comment:
the code block isn't rendering quite right - the tick marks are included in it. Same thing with the following code block. Not sure how much time we want to spend fixing it.